spec/features/admin_variant_details_edit_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'admin can edit variant details' do
+  let(:split_page) { app.admin_split_show_page }
+  let(:variant_page) { app.admin_variant_details_edit_page }
+
+  let!(:split) { FactoryGirl.create(:split, name: 'great_feature', registry: { enabled: 100 }) }
+
+  before do
+    login
+  end
+
+  it 'allows admins to edit variant details' do
+    split_page.load split_id: split.id
+    expect(split_page).to be_displayed
+
+    expect(split_page.variants.count).to eq 1
+    split_page.variants.first.edit_link.click
+
+    expect(variant_page).to be_displayed
+
+    variant_page.form.display_name.set 'Variant name'
```

Comment:
do we care about validating presence of these attributes via testing an empty form and submitting?